lib/routes/sign.js

```diff
@@ -96,15 +97,25 @@ module.exports = function (log, isA, error, signer, db, domain) {
             duration: duration,
             generation: sessionToken.verifierSetAt,
             lastAuthAt: sessionToken.lastAuthAt(),
-            verifiedEmail: sessionToken.email
+            verifiedEmail: sessionToken.email,
+            deviceId: sessionToken.deviceId ?
+              sessionToken.deviceId.toString('hex') : null
```

Comment:
That reminds me, I owe you a plan for ensuring most devices actually have a device id...